homeassistant/components/history_stats/data.py

```diff
@@ -54,6 +55,7 @@ def __init__(
         self._previous_run_before_start = False
         self._entity_states = set(entity_states)
         self._duration = duration
+        self._min_state_duration = min_state_duration
```

Comment:
Why not make this `self._min_state_duration = min_state_duration.total_seconds()` and get rid of `min_state_timestamp`?